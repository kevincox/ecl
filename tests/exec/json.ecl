num = 1
bool = false
dict = {
	nil_ = nil
	list = [1 2 3]
}
