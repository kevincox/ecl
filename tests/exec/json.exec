cmd: load json.ecl -fjson
