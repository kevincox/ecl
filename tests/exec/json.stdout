{
  "bool": false,
  "dict": {
    "list": [
      1.0,
      2.0,
      3.0
    ],
    "nil_": null
  },
  "num": 1.0
}
